cmd/mel-replay/db.go

```diff
@@ -88,6 +88,10 @@ func (d *DB) AncientRange(kind string, start, count, maxBytes uint64) ([][]byte,
 	return nil, errors.New("unimplemented")
 }
 
+func (d *DB) AncientBytes(kind string, id, offset, length uint64) ([]byte, error) {
+	return nil, errors.New("unimplemented")
+}
```

Comment:
why is this being introduced? https://github.com/OffchainLabs/nitro/blob/v3.9.0/cmd/mel-replay/db.go